styles.css

```diff
@@ -106,10 +115,410 @@ svg {
+  font-weight: 700;
+}
+
+.section1 ul,
+.background_wrapper ul {
+  display: flex;
+  list-style: none;
+  margin-top: 0.375em;
+  margin-left: -0.25em;
+}
+
+.section1 ul li,
+.background_wrapper ul li {
+  padding: 4px;
+}
+
+.section1 ul li button,
+.background_wrapper ul li button {
+  padding: 0.63em 0.75em;
+  border: 1px solid #8993a4;
+}
+
+/* styles section 2 to 7 */
+
+.background {
+  display: flex;
+  justify-content: center;
+}
+
+.background_wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+  width: 90%;
+  height: 100%;
+  background-image: url(img/background.png);
+}
+
+.background_wrapper .ul_li {
+  color: #fff;
+  background-color: rgba(255, 255, 255, 0.24);
+  margin-top: 20px;
+}
+
+.background_wrapper main,
+.background_wrapper section {
+  margin-left: 12px;
+}
+
+.background_wrapper h1 {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 32px;
+  color: #fff;
+  line-height: 44px;
+}
+
+.background_wrapper p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 1em;
+  line-height: 24px;
+  color: #fff;
+}
+
+.project {
+  width: 100%;
+  height: 3em;
+  background-color: var(--cardbutton);
+  border: none;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.project:hover {
+  background-color: #efa628fc;
+  border: 2px solid orangered;
+  cursor: pointer;
+}
+
+.project:active {
+  background-color: var(--cardbutton);
+}
+
+.project:disabled {
+  background-color: #8993a4;
+}
+
+.project a {
+  text-decoration: none;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+}
+
+/* styles for the about me section */
+.about {
+  display: flex;
+  justify-content: center;
+  background-color: #1c1a19;
+}
+
+.about_wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 6.3em;
+  width: 90%;
+  height: 100%;
+  background-repeat: no-repeat;
+  background-position: center right;
+}
+
+.about_wrapper .aboutme {
+  color: var(--cardbutton);
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+}
+
+.about_wrapper .aboutparagraph {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 1em;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.about_wrapper button {
+  background-color: var(--cardbutton);
+  width: 9.75em;
+  height: 3em;
+  border: none;
+  margin-top: 1.375em;
+  padding: 0.75em;
+}
+
+.about_wrapper button:hover {
+  background-color: #efa628fc;
+  border: 2px solid orangered;
+  cursor: pointer;
+}
+
+.about_wrapper button:active {
+  background-color: var(--cardbutton);
+}
+
+.about_wrapper button:disabled {
+  background-color: #8993a4;
+}
+
+.about_wrapper button a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.aboutimage {
+  display: flex;
+  justify-content: space-between;
+}
+
+.container {
+  /* background-color: blue; */
```

Comment:
- [ ] Kindly remove all commented pieces of code from your documents. This will make your work clean and more professional.